code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -896,6 +896,20 @@
 	affective_ap_range = 4
 	recoil = 4
 
+/obj/item/projectile/bullet/shotgun/biomatter //Unique niche round. High AP, Low damage, high agony. Good for mob crunching, or AP LTL uses
+	name = "biomatter slug"
+	damage_types = list(BURN = 10) // Thin little piece of biomass designed to defeat armor but not really large enough to cause super serious injuries.
+	agony = 40 // BURNS SO BAD
+	armor_penetration = 40 //high velocity
```

Comment:
this makes people in 100% space suit and robots now die to biomatter shot, this is also inconstant with other biomatter ammo

Reply:
It was a conscious design decision to make it work a little different. Rather than being a big fat expanding round, think more like a little sliver of biomass sharpened to a point and carried by a sabot.

Reply:
It still ultimately doesn't really have enough damage after modifiers to have any kinda serious usecase against robots